public/locales/en.json

```diff
@@ -17,6 +17,7 @@
   "connectWallet": "Connect Wallet",
   "currencyReserves": "Currency reserves",
   "dailyVolume": "Daily volume",
+  "deadline": "Transaction Deadline",
```

Comment:
add these keys with null values to `public/locales/zh-CN.json` so we can add translations later

Reply:
added deadline and minutes null values to Chinese localization